### PR TITLE
july dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-july-2023.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+Author: Kory Draughn
+Slug: irods-development-update-july-2023
+Status: published
+
+
+Hello Everyone,
+
+It's time for another monthly development update!
+
+To start, the interns are doing an excellent job on their projects. They are quickly steam-rolling through the issues. we now have an improved version of the zone report, a better Zone Management Tool, low-level examples of the XML protocol, and support for S3 PUT_SYNC in the Ingest tool. They will present their work at this upcoming TRiRODS meeting. Please join us.
```
Comment (trel) [inline content/posts/irods-development-update-july-2023.md:12]: Capital W

“We now have”

File: content/posts/irods-development-update-july-2023.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+
+We've exposed new options in **server_config.json** for controlling socket timeouts, delay server migration, etc. The C++ connection libraries now support alternative forms of authentication.
+
+The new iRODS CLI was presented at UGM. It now has partial support for many operations such as `ls`, `mv`, `get`, `put`, `touch`, etc.
+
+GenQuery2 is in a good position for the community to begin experimenting with it. We've added support for 4.2.11 and 4.2.12 so more users have access to it.
+
+We presented a new HTTP API for iRODS at UGM. This project is an experimental redesign of the C++ REST API. It's coming together really well. The goal is to release it on the heels of iRODS 4.3.1. Once released, users of the C++ REST API are encouraged to switch to the new HTTP API. 0.9.4 will be the final release of the C++ REST API.
+
+Python iRODS Client 1.1.9 is growing some new abilities. Work to make the PRC connect to the storage resource holding the replica for reading and writing is in progress. This upcoming release will also support auto-closing of data objects and the new authentication plugin framework.
```
Comment (trel) [inline content/posts/irods-development-update-july-2023.md:24]: “connect” -> “connect directly”

File: content/posts/irods-development-update-july-2023.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-july-2023.md]: Done

File: content/posts/irods-development-update-july-2023.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-july-2023.md]: Done

**Comment by korydraughn** [comment]
Should I squash?

Does the time at the top need to be changed? That's not reported AFAIK.

**Comment by trel** [comment]
Squash and set to noon

**Pull request merged**